lib/Backend/BackwardPass.cpp

```diff
@@ -160,7 +160,7 @@ BackwardPass::DoTrackNegativeZero() const
 bool
 BackwardPass::DoTrackBitOpsOrNumber() const
 {
-#if _WIN64
+#if defined(_WIN32) && defined(TARGET_64)
```

Comment:
Minor thing but I noticed that _WIN32 seems to be explicitly set to 1 sometimes, so maybe just check if _WIN32 && defined... ?